woocommerce/woocommerce
# Deploy payments banner in settings page

### Changes proposed in this Pull Request:



Closes #34166. This PR:
- Deploys payments banner experiment
- Removes experiment codes
- Adds a new track `wcadmin_settings_payments_banner_connect_click` for CTA
- Updated tests

### How to test the changes in this Pull Request:

1. Set up a new store in WCPay-available countries such as US
2. Open developer console and run `localStorage.setItem( 'debug', 'wc-admin:*' );` to set debug for tracks
3. Make sure "Preserve log" option is checked in developer console
4. Install WCPay plugin
5. Go to WooCommerce > Settings > Payment
6. Observe the payments banner is displayed
7. Click on "Get started"
8. Make sure it attempts to connect
9. Observe `wcadmin_settings_payments_banner_connect_click` track is called

### Other information:




### FOR PR REVIEWER ONLY:
